app/controllers/events/draft.js

```diff
@@ -0,0 +1,100 @@
+import Ember from 'ember';
+
+const { Controller, String } = Ember;
+export default Controller.extend({
+  columns: [
+    {
+      propertyName : 'name',
+      template     : 'components/ui-table/cell/cell-event',
+      title        : 'Name'
+    },
+    {
+      propertyName : 'startTime',
+      template     : 'components/ui-table/cell/cell-date',
+      title        : 'Date'
+    },
+    {
+      propertyName : 'roles',
+      template     : 'components/ui-table/cell/cell-roles',
+      title        : 'Roles'
+    },
+    {
+      propertyName : 'sessions',
+      template     : 'components/ui-table/cell/cell-sessions',
+      title        : 'Sessions'
+    },
+    {
+      propertyName : 'speakers',
+      title        : 'Speakers'
+    },
+    {
+      propertyName : 'tickets',
+      template     : 'components/ui-table/cell/cell-tickets',
+      title        : 'Tickets'
+    },
+    {
+      propertyName : 'url',
+      title        : 'Public URL',
+      class        : 'hide'
+    },
+    {
+      template : 'components/ui-table/cell/cell-buttons',
+      title    : 'Actions'
+    }
+
+  ],
+  data: [{
```

Comment:
will be done by [#155](https://github.com/fossasia/open-event-frontend/pull/155)